frontend/src/components/App/App.tsx

```diff
@@ -1,32 +1,40 @@
 import React, { useRef } from 'react'
 import { Route, Routes } from 'react-router-dom'
+import axios from 'axios'
+import Cookies from 'js-cookie'
 import './App.css'
 
 import BikeGallery from '../BikeGallery/BikeGallery'
 import BikeDetails from '../BikeDetails/BikeDetails'
-import Header from '../Header/Header'
+import SignUp from '../SignUp/SignUp'
+import Login from '../Login/Login'
 
-import useBikesApi from '../../hooks/useBikesApi'
-import AddBike from '../AddBike/AddBike'
+// @ts-ignore
```

Comment:
This line looks a bit like an indicator for not optimal code, is there no way to change you code to work without ignoring type errors?

Reply:
👍